cobra/io/sbml.py

```diff
@@ -48,12 +47,19 @@
 from cobra.util.solver import linear_reaction_coefficients, set_objective
 
 
+try:
+    from cStringIO import StringIO  # Python 2
+except ImportError:
+    from io import StringIO
+
+
 class CobraSBMLError(Exception):
     """ SBML error class. """
     pass
 
 
 LOGGER = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
```

Comment:
A library package should never configure the logging or if we do decide to do so since we communicate important INFO level messages then the `sbml.py` is not the place for it.

```suggestion
```